app/extras/queries/visible_motions.rb

```diff
@@ -7,7 +7,7 @@ def initialize(user: nil, groups: nil, group_ids: nil)
       group_ids = Array(groups).map(&:id)
     end
 
-    @relation = Motion.joins(:discussion => :group).merge(Group.published).preload(:discussion)
+    @relation = Motion.joins(discussion: :group).where('groups.archived_at IS NULL').preload(:discussion)
```

Comment:
That's a little bit sad but I guess you gotta do it and move on. 

Seems weird though, running those lines in the console makes it look easy. However the rest of the query does get a bit scary sometimes.